FanartHandler/FanartSelectedOther.cs

```diff
@@ -151,26 +155,10 @@ public void RefreshGenericSelectedProperties(Utils.SelectedType property, ref st
             }
             AddSelectedGenreProperty(SelectedGenre, SelectedItem, property);
 
-            // Trakt remove ClearArt for pages
-            var window = GUIWindowManager.GetWindow(Utils.iActiveWindow);
-            if (window != null)
+            if (isVideo)
             {
-                var selectedListItem = GUIControl.GetSelectedListItem(Utils.iActiveWindow, window.GetFocusControlId());
-                if (selectedListItem != null)
-                {
-                    if (selectedListItem.IsFolder)
-                    {
-                        IsSelectedVideo = true;
-                        ClearSelectedMoviePropertys();
-                    }
-                    else
-                    {
-                        if (isVideo)
-                        {
-                            AddSelectedMoviePropertys();
-                        }
-                    }
-                }
+                Console.WriteLine(Utils.iActiveWindow.ToString());

```

Comment:
It is better to use the Log output rather than the console for advanced debugging.
```
          if (Utils.AdvancedDebug)
          { 
            logger.Debug(".........
          }
```